dirty.html

```diff
@@ -1,205 +1,218 @@
+<!DOCTYPE html>
 <html>  
+  <!-- start of head -->
   <head>
-    <h1>WDI London September 2013
+    <title>WDI London September 2013</title>
     <meta value="description" content="Web development immersive ">
   </head>
-  <div class="body">
-    <div>
-        <div>
+  <!-- end of head -->
```

Comment:
All these `<!-- irrelevant comments -->` should be removed as they don't really add more info that you would immediately get by looking at the code :)